packages/fether-react/src/stores/sendStore.js

```diff
@@ -75,6 +87,41 @@ export class SendStore {
     });
   };
 
+  /**
+   * Send a raw, signed transaction.
+   */
+  sendRaw = () => {
+    debug('Sending raw tx.', this.tx.rawSigned);
+
+    return new Promise((resolve, reject) => {
+      postRaw$(this.tx.rawSigned).subscribe(txStatus => {
+        if (txStatus.signed) {
+          this.listenForConfirmations();
+          resolve();
+        }
+        this.setTxStatus(txStatus);
+        debug('Tx status updated.', txStatus);
+      }, reject);
+    });
+  };
+
+  /**
+   * Get the RLP of an (unsigned) transaction.
+   */
+  getRlp = () => {
+    return transactionToRlp(this.tx);
+  };
+
+  /**
+   * Sign the transaction with the given signature.
+   */
+  @action
+  signRaw = signature => {
+    const signed = signTransactionWithSignature(this.tx, signature);
```

Comment:
Hmm, so `postRaw$('0x' + signature)` won't work?

Reply:
hmm, not sure I understand
signature is just the signature, postRaw$ needs to be provided with a signed tx (i.e. tx+sig)
https://ethereum.stackexchange.com/a/2097
to my understanding: signature is serialized v, s, r; postRaw$ needs serialized nonce, gasprice, startgas, to, value, data, v, r, s

Reply:
Ah, right, I thought Parity Signer did the rlp(tx + signature) part, my bad!